src/components/board/index.jsx

```diff
@@ -17,6 +17,23 @@ export default function Board({ columns, rows, guesses, evaluations }) {
         );
     }
 
+    function ActiveRow({ active }) {
```

Comment:
There's assuredly some DRY we can do to unify "Guess" with "Active" and "Empty" in the future.